pkg/pubsub/node_data.go

```diff
@@ -164,7 +164,7 @@ func (n *NodeData) Joined(nodeVersion string) {
 
 	logMessage := fmt.Sprintf("[+] %s node joined: %s", map[bool]string{true: "Staked", false: "Unstaked"}[n.IsStaked], n.MultiaddrsString)
 	if n.IsStaked {
-		logrus.Info(logMessage)
+		logrus.Debug(logMessage)
```

Comment:
Why this change? I think it does make sense to leave it as `Info` for staked nodes. OTOH if we leave it as `Debug`, the `if` is pointless.